R/util.R

```diff
@@ -335,11 +335,25 @@ same_type <- function(vector, original){
   } else if(is.integer(original)){
     as.integer(vector)
   } else if(inherits(original, "Date")) {
-    # when original data is Date
-    as.Date(vector, tz = lubridate::tz(original))
+    # when original data is Date.
+    # if the column is wrapped with lubridate funciton, it's possible that data is converted to numeric like year
```

Comment:
Typo: funciton

Reply:
Not too sure why in the first place this function has to be called in such case. Isn't just simply not calling it an option?